MVC/application/view/components/header.php

```diff
@@ -0,0 +1,25 @@
+<link rel="stylesheet" href="<?php echo BASEURL; ?>/assets/css/style.css?version">
```

Comment:
Please add a bit of comment about the functionality of this page

Reply:
I will add comments later.